faster_whisper/vad.py

```diff
@@ -35,8 +28,9 @@ class VadOptions(NamedTuple):
       speech_pad_ms: Final speech chunks are padded by speech_pad_ms each side
     """
 
-    threshold: float = 0.5
-    min_speech_duration_ms: int = 250
+    onset: float = 0.5
+    offset: float = onset - 0.15
+    min_speech_duration_ms: int = 0
```

Comment:
Can you maybe leave these options (threshold, onset, offset) as they were, e.g. not rename them as it would break signature & parameter passing APIs?

Why are you changing min_speech_duration_ms to 0? I think 250ms is a sane default otherwise you may end up with segments that are very small for having speech inside, maybe even empty ones?

Reply:
it's best to give the users the freedom to tune the parameters as they wish, previously `offset` was fixed to `threshold - 0.15`, but now users have the option to tune it as they with without having to play with the code internals, it might not be backwards compatible but it's a very minimal change to adapt

as for `min_speech_duration_ms`, benchmarks (YT Commons and Librispeech) showed that dropping it from 250 to 0 had minimal positive or no effect on sequential inference, but it had a very positive impact on batched inference as it combines segments differently than the sequential